package.json

```diff
@@ -38,7 +38,8 @@
   "dependencies": {
     "autosize": "3.0.20",
     "base64-image-loader": "1.2.0",
-    "flatpickr": "2.4.7"
+    "flatpickr": "2.4.7",
+    "moment": "2.18.1"
```

Comment:
we shoudn't need to use `moment` - it is a bloated dependency that gets even larger when adding i18n support, and using native `Date` tools + react-intl's [Date formatting components](https://github.com/yahoo/react-intl/wiki/Components#date-formatting-components) can usually get us where we need to be.

`moment.js` makes things easy, but at a pretty significant cost.